deploy/data_exporter/data_export_function/app.py

```diff
@@ -0,0 +1,178 @@
+import os
+import subprocess
+from datetime import datetime, timedelta, timezone
+
+import boto3
+import psycopg
+from psycopg import sql
+
+ssm = boto3.client("ssm")
+s3 = boto3.client("s3", region_name="eu-west-1")
+bucket_name = "dc-ynr-short-term-backups"
+current_time = datetime.now().isoformat()
+PREFIX = "ynr-export"
+FILENAME = f"{PREFIX}-{current_time.replace(':', '-')}.dump"
+
+
+def get_parameter(name):
+    response = ssm.get_parameter(Name=name)
+    return response["Parameter"]["Value"]
+
+
+SOURCE_DATABASE = "ynr"
+TMP_DATABASE_NAME = "ynr-for-dev-export"
+DB_HOST = get_parameter("/ynr/production/POSTGRES_HOST")
+DB_USER = get_parameter("/ynr/production/POSTGRES_USERNAME")
+DB_PASSWORD = get_parameter("/ynr/production/POSTGRES_PASSWORD")
+DB_PORT = "5432"
+os.environ["PGPASSWORD"] = DB_PASSWORD
+
+
+def get_db_conn(db_name):
+    conn = psycopg.connect(
+        dbname=db_name,
+        user=DB_USER,
+        password=DB_PASSWORD,
+        host=DB_HOST,
+        port=DB_PORT,
+    )
+    conn.autocommit = True
+    return conn
+
+
+def create_database_from_template():
+    # Connect to the PostgreSQL server (usually to the 'postgres' database for administrative tasks)
+    conn = get_db_conn(SOURCE_DATABASE)
+    # Enable autocommit to run CREATE DATABASE commands
+    try:
+        with conn.cursor() as cur:
+            print(f"Deleting {TMP_DATABASE_NAME}")
+            cur.execute(
+                sql.SQL("DROP DATABASE IF EXISTS {};").format(
+                    sql.Identifier(TMP_DATABASE_NAME)
+                )
+            )
+        with conn.cursor() as cur:
+            # SQL to create the new database from the template
+            print(f"Creating {TMP_DATABASE_NAME}")
+            cur.execute(
+                sql.SQL("CREATE DATABASE {} TEMPLATE {};").format(
+                    sql.Identifier(TMP_DATABASE_NAME),
+                    sql.Identifier(SOURCE_DATABASE),
+                )
+            )
+            print(
+                f"Database '{TMP_DATABASE_NAME}' created successfully from template '{SOURCE_DATABASE}'."
+            )
+    except psycopg.Error as e:
+        print(f"Error creating database: {e}")
+    finally:
+        conn.close()
+
+
+def clean_database():
+    conn = get_db_conn(db_name=TMP_DATABASE_NAME)
+    with conn.cursor() as cur:
+        print("Cleaning Users table")
+        cur.execute(
+            """UPDATE auth_user SET 
+                email = CONCAT('anon_', id, '@example.com'), 
+                password = md5(random()::text);
+            """
+        )
+        print("Cleaning Account email table")
+        cur.execute(
+            """UPDATE auth_user SET 
+                email = CONCAT('anon_', id, '@example.com');
```

Comment:
You already did this on line 79

Reply:
I'd also propose that we might leave all @democracyclub.org.uk accounts or all admin accounts alone here, just for convenience of being able to log in with your own creds.

Reply:
I'm not convinced about that...For a start, having other DC people's (hashed) password isn't awesome (but better than every users). But also, these dumps might be given to 3rd parties working on the project at some point. 

Running `./manage.py changepassword` isn't a big deal, but also now we have OTT login on YNR you can just log in by  entering your fake email address.

Reply:
OK. One thing I'd slightly lost sight of is that we have a username which is distinct from email, so I will still be able to identify which user account is "mine" by username even if we nuked all the emails.